docsys/src/main/node/docsys/update.sh

```diff
@@ -2,4 +2,11 @@
 rm -rf dist .nuxt
 npm run generate
 rm -rf ../../resources/docsys-guidebook
-mv dist ../../resources/docsys-guidebook
+cp -r dist ../../resources/docsys-guidebook
```

Comment:
This works for now, but we have multiple doc sources that are all loaded automatically in the run time.
A future version of our static site generation should use `nuxt generate` with an optional environment variable that tells the dynamic route mechanism how to find and cache all the pages exposed by the running docserver.

We can get by for now by updating this script, but the number of `../` is concerning.

Reply:
works for me